apprentice-react-o-shop/src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import '../../styles/bootstrap.min.css'
```

Comment:
Let's get together and talk about finding a better way to add Bootstrap. Right now this includes that long css file and is only available to this component. We want to mention it once and have it available to the entire project.